monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,115 @@
+[id="configuring-the-monitoring-stack"]
+= Configuring the monitoring stack
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: configuring-the-monitoring-stack
+
+toc::[]
+
+The {product-title} 4 installation program provides only a low number of configuration options before installation. Configuring most {product-title} framework components, including the cluster monitoring stack, happens post-installation.
+
+This section explains what configuration is supported, shows how to configure the monitoring stack, and demonstrates several common configuration scenarios.
+
+== Prerequisites
+
+* The monitoring stack imposes additional resource requirements. Consult the computing resources recommendations in link:https://docs.openshift.com/container-platform/4.6/scalability_and_performance/scaling-cluster-monitoring-operator.html[Scaling the Cluster Monitoring Operator] and verify that you have sufficient resources.
+
+// Maintenance and support for monitoring
+include::modules/osd-monitoring-maintenance-and-support.adoc[leveloffset=+1]
+include::modules/osd-monitoring-support-considerations.adoc[leveloffset=+2]
+include::modules/osd-monitoring-unmanaged-monitoring-operators.adoc[leveloffset=+2]
+
+// Configuring the monitoring stack
+include::modules/osd-monitoring-configuring-the-monitoring-stack.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc#accessing-monitoring-for-user-defined-projects[Accessing monitoring for user-defined projects]
+
+// Configurable monitoring components
+include::modules/osd-monitoring-configurable-monitoring-components.adoc[leveloffset=+1]
+
+// Moving monitoring components to different nodes
+include::modules/osd-monitoring-moving-monitoring-components-to-different-nodes.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Accessing monitoring for user-defined projects]
+* link:https://docs.openshift.com/container-platform/4.6/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes]
```

Comment:
This is a link that directs the reader to the Nodes book OpenShift Container Platform documentation. Is this relevant to ROSA, or should this link be removed? The reader might not realise that they are being directed to the docs for a different platform type.

Reply:
Yes, they're supposed to access OCP docs for that. Thanks for checking.